client/app/achievements/achievements.html

```diff
@@ -11,34 +11,11 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```

Comment:
moves the sidebar to the top of the page on mobile![screenshot 2016-09-27 at 12 31 33 pm](https://cloud.githubusercontent.com/assets/2281120/18882708/5d2c9aac-84ae-11e6-95a6-7e7ad63af549.png)